project.mml

```diff
@@ -725,6 +726,8 @@ Layer:
                 END AS int_surface,
                 CASE WHEN access IN ('destination') THEN 'destination'::text
                   WHEN access IN ('no', 'private') THEN 'no'::text
+                  -- highway=busway is access=no/bus=designated by default.
```

Comment:
Since this style provides important feedback to mappers, we generally try to avoid giving the impression that a tag exists if it does not. Please remove the default `access=no` rendering

Reply:
The tag is [explicitly documented](https://wiki.openstreetmap.org/wiki/Tag:highway%3Dbusway) as implying `access=no` (and `bus=designated`) if no access tags are set. Should Carto really redefine a tag? Providing feedback means reporting what the effect of the tags set is. That means that:

```
highway=busway
```

and

```
highway=busway
access=no
```

Should render the same, doesn't it?

Reply:
Also note that this line is not about rendering of course, but about setting that default value to its documented state. Whether to act on that in terms of rendering is not decided here in the query.